README.md

```diff
@@ -209,6 +210,21 @@ HealthMonitor.configure do |config|
 end
 ```
 
+### Adding a Warning threshold
+If you want to be alerted to something concerning, but don't consider it an error you can raise an optional inherited exception to get back a warning:
```

Comment:
I'd simplify this copy a little bit: "In the case you don't want to raise an error, but only show an warning status - make sure to raise a `HealthMonitor::Warning` instead of `HealthMonitor::Error`".